R/covr.R

```diff
@@ -203,6 +203,9 @@ package_coverage <- function(path = ".",
   # add hooks to the package startup
   add_hooks(pkg$package, tmp_lib)
 
+  if (should_enable_parallel_mcexit_fix(pkg))
```

Comment:
I would prefer to put the conditional directly in `add_hooks()` rather than defining a new function to do this and replicating the read a write lines.